tools/distrib/buildozer.sh

```diff
@@ -30,11 +30,12 @@ function error_handling() {
 }
 
 function download_buildozer() {
-    platform="$(uname -s)"
+    platform="$(uname -sm)"
     case "${platform}" in
-        Linux*)     download_link="https://github.com/bazelbuild/buildtools/releases/download/${BUILDOZER_VERSION}/buildozer-linux-amd64";;
-        Darwin*)    download_link="https://github.com/bazelbuild/buildtools/releases/download/${BUILDOZER_VERSION}/buildozer-darwin-amd64";;
-        *)          error_handling "Unsupported platform: ${platform}";;
+        "Linux x86_64")     download_link="https://github.com/bazelbuild/buildtools/releases/download/${BUILDOZER_VERSION}/buildozer-linux-amd64";;
```

Comment:
Please do the same for `Linux arm64`, since there's a `buildozer-linux-arm64` release as well.